### ISSUE TITLE
Stray println! in Storage bidi read worker error path

### ISSUE THREAD

**Issue by coryan** [googleapis/google-cloud-rust#6692]
Unconfirmed.  This is the result of a search with Claude, may be a false positive.

A stray `println!` statement remains in production library code in `src/storage/src/storage/bidi/worker.rs`:

https://github.com/googleapis/google-cloud-rust/blob/f02b42c2ac5720334cd2b5335824bd4166264527/src/storage/src/storage/bidi/worker.rs#L104

```rust
println!("worker closed with error: {e:?}");
```

When a bidi worker encounters an error, this writes directly to the hosting application's `stdout` on every subsequent `read_range()` invocation instead of utilizing standard `tracing` macros.
